src/nodule.js

```diff
@@ -17,16 +12,22 @@ export default class Nodule {
         this.loaders = loaders || [];
         this.scope = scope;
 
+        // Declassified loaders contain configuration that does not contain any secrets
+        this.declassifiedLoaders = [];
```

Comment:
Calling this `declassified` over `private` as private has a different meaning in CS